src/diffusers/utils/hub_utils.py

```diff
@@ -48,6 +50,9 @@
 HUGGINGFACE_CO_TELEMETRY = HUGGINGFACE_CO_RESOLVE_ENDPOINT + "/api/telemetry/"
 
 
+_CACHED_NO_EXIST = object()
```

Comment:
```suggestion
from huggingface_hub.file_download import _CACHED_NO_EXIST
```

If you want to use this object, it has to be imported from `file_download`.